moment_kinetics/src/reference_parameters.jl

```diff
@@ -61,7 +61,12 @@ function setup_reference_parameters(input_dict, warn_unexpected::Bool)
 end
 
 """
-Calculate normalized ion-ion collision frequency at reference parameters for Coulomb collisions.
+Calculate normalized ion-ion collision frequency for Coulomb collisions using reference
+density, mass and speed.
+Note this is not the thermal collision frequency evaluated with reference parameters, but
+rather the quantity that when multiplied by \$\\hat{n}s/v_{Ts}^3\$ gives the
+de-dimensionalised thermal collision frequency - the difference is a factor of
+\$2^{3/2}\$.
 
 Currently valid only for hydrogenic ions (Z=1)
```

Comment:
Not directly related to the changes here, but since the docs are changed above, it is worth noting that I only envisage calculating the result here for hydrogen and multiplying and dividing by appropriate factors of Z (charge #) and A (mass) in the `fokker_planck.jl` to construct the correct prefactor for the operator. If we were to try to include the variation of the Coulomb logarithm, we would need to do more.